stdlib/test_packages.go

```diff
@@ -4,34 +4,38 @@ package stdlib
 
 import (
 	ast "github.com/influxdata/flux/ast"
+	date "github.com/influxdata/flux/stdlib/date"
 	experimental "github.com/influxdata/flux/stdlib/experimental"
 	http "github.com/influxdata/flux/stdlib/http"
 	monitor "github.com/influxdata/flux/stdlib/influxdata/influxdb/monitor"
 	secrets "github.com/influxdata/flux/stdlib/influxdata/influxdb/secrets"
 	v1 "github.com/influxdata/flux/stdlib/influxdata/influxdb/v1"
 	promql "github.com/influxdata/flux/stdlib/internal/promql"
+	regexp "github.com/influxdata/flux/stdlib/regexp"
 	strings "github.com/influxdata/flux/stdlib/strings"
 	chronograf "github.com/influxdata/flux/stdlib/testing/chronograf"
 	kapacitor "github.com/influxdata/flux/stdlib/testing/kapacitor"
 	pandas "github.com/influxdata/flux/stdlib/testing/pandas"
-	testdata "github.com/influxdata/flux/stdlib/testing/testdata"
+	promql1 "github.com/influxdata/flux/stdlib/testing/promql"
```

Comment:
Why are there two promql packages?

Reply:
There's `internal/promql` which involves tests of those functions from that package. I decided instead of attempting to merge things, I would instead just put the transpiler tests into testing as "integration tests". Since this was a structure change, I didn't want to also make more decisions than I needed to at this moment.

Reply:
I see those are transpiler tests. Got it.